frontend/src/scenes/notebooks/Nodes/NotebookNodeTimestamp.tsx

```diff
@@ -0,0 +1,90 @@
+import { mergeAttributes, Node, NodeViewProps } from '@tiptap/core'
+import { NodeViewWrapper, ReactNodeViewRenderer } from '@tiptap/react'
+import { NotebookNodeType } from '~/types'
+import { sessionRecordingPlayerLogic } from 'scenes/session-recordings/player/sessionRecordingPlayerLogic'
+import { dayjs } from 'lib/dayjs'
+import { JSONContent } from '../Notebook/utils'
+import { sessionRecordingPlayerProps } from './NotebookNodeRecording'
+import clsx from 'clsx'
+import { lastChildOfType } from '../Notebook/Editor'
+
+const Component = (props: NodeViewProps): JSX.Element => {
+    const playbackTime: number = props.node.attrs.playbackTime
+
+    return (
+        <NodeViewWrapper as="span" class={clsx('Timestamp', props.selected && 'Timestamp--selected')}>
+            {formatTimestamp(playbackTime)}
+        </NodeViewWrapper>
```

Comment:
Maybe more specific? `NotebookRecordingTimestamp` or something.

Timestamp feels to generic and could end up clashing with something else I would imagine...